src/main/java/pro/sky/Coursework2/controller/ExamController.java

```diff
@@ -0,0 +1,28 @@
+package pro.sky.Coursework2.controller;
+
+import pro.sky.Coursework2.model.Question;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+
+import pro.sky.Coursework2.service.ExaminerServiceImpl;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/get")
+public class ExamController {
+    private final ExaminerServiceImpl examinerService;
+
+
+    public ExamController(ExaminerServiceImpl examinerService) {
```

Comment:
```suggestion
    public ExamController(ExaminerService examinerService) {
```
Инжектить надо по интерфейсу всегда, а не по реализации